readme.md

```diff
@@ -1,21 +1,141 @@
-# Safari Webstore Upload
+# Safari Web Extension Publisher
 
-This module does cool stuff with the Safari Webstore.
+This module uses Apple's `safari-web-extension-converter` to convert a Web Extension (i.e. Chrome Extension) to a Safari Web Extension, and uploads the native bundle to the Apple App Store using Fastlane. Deployment is available for both iOS and macOS platforms. Runs on both macOS and GitHub-hosted macOS runners. This is the Safari implementation for [Plasmo Browser Platform Publisher](https://github.com/PlasmoHQ/bpp).
 
 Feature includes:
-
-- TypeScript, with type exports
+- TypeScript API, with type exports
+- Pure ECMAScript module
 - Frozen dependencies, updated via renovatebot
+- Support for GitHub-hosted macOS runners
+
+Limitations: 
+- Only supports Manifest V2 extensions (for now)
```

Comment:
I recall to support MV3, we will need to ditch XCode's extension converter, and use some of the community made one?